### ISSUE TITLE
Ubuntu DEB packaging: can't start after reboot due to absence of /var/run/3proxy folder

### ISSUE THREAD

**Issue by SuperBoss9** [3proxy/3proxy#739]
It is strange situation. Ubuntu 20.04 LTS with 3proxy installed from deb experimental package.
It can't start until I've create /vat/run/3proxy folder manually.

After the system reboot this folder is deleted (somehow) and 3proxy is not able to start.

What shall I do with this?

**Comment by SuperBoss9** [comment]
So, the init script should be changed in order to add some mkdir commands. Or pid file moved just into /var/run folder.
I did the second as a workaround. It works.

**Comment by z3APA3A** [comment]
Duplicate of https://github.com/3proxy/3proxy/issues/674, should be fixed in current master (will be released with 0.9.5)
